images/kubectl-build-deploy-dind/build-deploy-docker-compose.sh

```diff
@@ -933,16 +941,71 @@ if [[ "${CAPABILITIES[@]}" =~ "backup.appuio.ch/v1alpha1/Schedule" ]]; then
   fi
   TEMPLATE_PARAMETERS+=(-p BAAS_BUCKET_NAME="${BAAS_BUCKET_NAME}")
 
-  # Run Backups every day at 2200-0200
-  BACKUP_SCHEDULE=$( /kubectl-build-deploy/scripts/convert-crontab.sh "${NAMESPACE}" "M H(22-2) * * *")
+  # Pull in .lagoon.yml variables
+  LAGOON_YAML_KEYS=($(cat .lagoon.yml | shyaml keys))
+  if containsElement "production-backup-retention" "${LAGOON_YAML_KEYS[@]}"; then
+    LAGOON_YAML_BACKUP_RETENTION_KEYS=($(cat .lagoon.yml | shyaml keys production-backup-retention))
+
+    if containsElement "monthly" "${LAGOON_BACKUP_RETENTION_YAML_KEYS[@]}"; then
+      PRODUCTION_MONTHLY_BACKUP_RETENTION=$(cat .lagoon.yml | shyaml keys production-backup-retention.monthly -q)
+    else
+      PRODUCTION_MONTHLY_BACKUP_RETENTION=""
+    fi
+
+    if containsElement "weekly" "${LAGOON_BACKUP_RETENTION_YAML_KEYS[@]}"; then
+      PRODUCTION_WEEKLY_BACKUP_RETENTION=$(cat .lagoon.yml | shyaml keys production-backup-retention.weekly -q)
+    else
+      PRODUCTION_WEEKLY_BACKUP_RETENTION=""
+    fi
+
+    if containsElement "daily" "${LAGOON_BACKUP_RETENTION_YAML_KEYS[@]}"; then
+      PRODUCTION_DAILY_BACKUP_RETENTION=$(cat .lagoon.yml | shyaml keys production-backup-retention.daily -q)
+    else
+      PRODUCTION_DAILY_BACKUP_RETENTION=""
+    fi
+  else
+    PRODUCTION_MONTHLY_BACKUP_RETENTION=""
+    PRODUCTION_WEEKLY_BACKUP_RETENTION=""
+    PRODUCTION_DAILY_BACKUP_RETENTION=""
+  fi
+
+  # Pull in environment type (development/production)
+  LAGOON_ENVIRONMENT_TYPE=${ENVIRONMENT_TYPE}
```

Comment:
why define a new variable if there is already one? 
let's just use `ENVIRONMENT_TYPE` directly

Reply:
Makes sense, will do.